.github/workflows/main.yml

```diff
@@ -91,9 +91,19 @@ jobs:
 
     - name: Test with pytest and run coverage
       run: |
-        coverage run -m pytest tests/
+        coverage run -m pytest tests/fortex/health/processors/icd_coding_processor_test.py
+        coverage run -m pytest tests/fortex/health/processors/negation_context_analysis_test.py
```

Comment:
we can't just test one or two test cases, let's use a matrix strategy to test different processors separately.